lib/common/options.c

```diff
@@ -460,15 +460,6 @@ static const pcmk__cluster_option_t cluster_options[] = {
             "configuration"),
         NULL,
     },
-    {
-        // @COMPAT Disallowed by schema
-        PCMK__OPT_REMOVE_AFTER_STOP, NULL, PCMK_VALUE_BOOLEAN, NULL,
-        PCMK_VALUE_FALSE, pcmk__valid_boolean,
-        pcmk__opt_schedulerd|pcmk__opt_deprecated,
-        N_("Whether to remove stopped resources from the executor"),
-        N_("Values other than default are poorly tested and potentially "
-            "dangerous."),
-    },
```

Comment:
also doc/sphinx/Pacemaker_Explained/cluster-options.rst

Reply:
whoops, had these in separate commits but they were next to the schema commits, not the feature commits